### PR TITLE
[chart]  Add storage class annotation and label handling

### PR DESCRIPTION
**Pull request by nicholasmhughes**
**Is this a bug fix or adding new feature?**
Depends on your point of view... :wink:   The issue is categorized as a bug.

**What is this PR about? / Why do we need it?**
Adds storage class annotation and label handling to the chart. See #733 

**What testing is done?**

### CODE REVIEW

File: charts/aws-ebs-csi-driver/values.yaml [hunk 1]
```diff
@@ -109,6 +109,9 @@ serviceAccount:
 storageClasses: []
 # Add StorageClass resources like:
 # - name: ebs-sc
+#   # annotation metadata
+#   annotations:
+#     storageclass.kubernetes.io/is-default-class: "true"
```
Comment (ayberk) [inline charts/aws-ebs-csi-driver/values.yaml:114]: Did we miss the "labels" field here? It's omitted on the storageclass.yaml.

**Review by ayberk — COMMENTED** [review]
/ok-to-test

File: charts/aws-ebs-csi-driver/values.yaml (hunk 1 above)
Comment (nicholasmhughes) [reply charts/aws-ebs-csi-driver/values.yaml:114]: I handled it in the template. I can't think of a great reason to have a label vs and annotation, so I didn't include it in the sample. I can if required.

Comment (ayberk) [reply charts/aws-ebs-csi-driver/values.yaml:114]: That'd be nice. Documenting all fields in the example could save operators from reading the template, so I think it's worth doing.

Comment (nicholasmhughes) [reply charts/aws-ebs-csi-driver/values.yaml:114]: @ayberk I added the label example. Let me know if that looks ok.

**Comment by nicholasmhughes** [comment]
@ayberk is there anything else I need to do for this PR?

**Comment by ayberk** [comment]
@nicholasmhughes If you can squash to a single commit I can merge it. Thanks a lot!

**Comment by nicholasmhughes** [comment]
@ayberk is that good enough?

**Comment by ayberk** [comment]
/lgtm
/approve

Thanks!

**Comment by nicholasmhughes** [comment]
thank you!

**Pull request merged**